_test/test_admin/test_pc_spec_config.m

```diff
@@ -85,8 +121,21 @@ function test_set_pc_type(obj)
             cm.this_pc_type = 'a_mac';
             assertEqual(cm.this_pc_type,'a_mac');
             assertEqual(cm.pc_config_num,3);
+            
+            kpc = cm.known_pc_types;
+            nconf = numel(kpc);
+            for i=1:nconf
+                cm.this_pc_type = i;
+                assertEqual(cm.this_pc_type,kpc{i});
+                assertEqual(cm.pc_config_num,i);
+            end
+            for i=1:nconf
+                cm.this_pc_type = kpc{i};
+                assertEqual(cm.this_pc_type,kpc{i});
```

Comment:
this test is confirming the previous line has executed:
```
a = b
assertEqual(a, b)
```

Reply:
this line checks if setters worked correctly. There are no abbreviation logic added to the code yet , (like its done in config clasees) as I do not think, it is important enough, but if it does one day, the test is there.